### DISCUSSION TITLE
Interest in Learning Analytics for Reveal.js presentations

### DISCUSSION THREAD

**Discussion by FaIng95** [hakimel/reveal.js#3244] [General]
Hello to all my fellow Reveal.js enthusiasts, 

I am currently working on my master thesis at the university of Muenster (Germany) to integrate web tracking of Reveal.js presentations with learning analytics approaches to give lecturers further information on how exactly their slides are used by students and which aspects might need improvement.

Therefore, I’m looking for interview partners, who use Reveal.js in a teaching context and are interested in a possible integration of learning analytics. @Jmuccigr, @finnito, @axiomtutor, @egrosclaude, @mbotsch, @seefood, In recent issues and discussions you mentioned using Reveal.js in a lecture context and I hope some of you might be willing to participate in a short interview regarding an integration of learning analytics. 

Of course, everyone else using Reveal.js for their lectures and wanting to participate in the interviews is also very much welcome and appreciated.

Please feel free to email me directly (public email is exposed in profile) or join this discussion, if you are interested in the subject.

Best regards and thanks in advance!

**Comment by seefood** [comment]
I never taught students with it, I just gave a few talks at local club meetings (Linux system, DevOps, Skepticism, Sci-fi, popular science and the like) and I always include a link in letters and a QR code to my personal website where I keep the latest copy of the slides, though I never looked for the stats in my reports.

**Comment by mbotsch** [comment]
I regularly use Reveal-based slides for teaching at TU Dortmund University on Bachelor and Master level. We actually use our tool [decker](https://gitlab2.informatik.uni-wuerzburg.de/decker/decker), which extends [pandoc](https://pandoc.org/) for converting Mardown to Reveal slides and extends Reveal through plugins designed for professional teaching ("infinite" whiteboard, audience response system, posting questions, video recording, etc).

I would very likely not include learning analytics, since any kind of tracking will make students skeptical, but I'm happy to talk about it. Just send me an email (Mario Botsch, TU Dortmund) and we can make an appointment.

**Comment by finnito** [comment]
Kia ora @EdgarKnusper!

I teach high school-level Science, Geology, and Physics with RevealJS on a daily basis. I use `pandoc` to generate the slides which are then incorporated into my website (https://putaiao.nz). I use the slides as a source of notes for the students and to help structure and guide the lesson. The students review the slides in their own time to help them revise, do homework or catch up when they are absent (or at least I like to think they do).

A good example of my slides are the Doppler Effect notes for my Year 12 Physics students here: https://putaiao.nz/12phy/as91523/slides/doppler-effect.html.

Tracking is an interesting concept - it could be interesting to see some stats on how often they are opened, etc. Feel free to email me lsf@cashmere.school.nz if this sounds like a suitable context.

**Comment by peter-lyons-kehl** [comment]
Did anyone find any specific mainstream/public analytics (whether for learning, or analytics in general) more suited to Reveal.js (than other analytics)?
